pandas/tests/sparse/test_series.py

```diff
@@ -1360,4 +1360,4 @@ def test_numpy_func_call(self):
                  'argmin', 'argmax', 'min', 'max']
         for func in funcs:
             for series in ('bseries', 'zbseries'):
-                getattr(np, func)(getattr(self, series))
+                getattr(np, func)(getattr(self, series)._values)
```

Comment:
Using the series object itself causes a warning to be thrown here. Since we are just calling numpy functions on the Series object anyway, if we call the underling series._values, then the warning is not thrown.